### PR TITLE
Downgrade INFO logs to just that

### PR DESCRIPTION
**Pull request by gtmills**
BMCWEB_LOG_INFO vs BMCWEB_LOG_CRITICAL.

Fixes 613773

### CODE REVIEW

**Review by baemyung — APPROVED** [review]
I agree to change those logs as INFO.

File: include/dump_offload.hpp [hunk 1]
```diff
@@ -185,8 +185,8 @@ class Handler : public std::enable_shared_from_this<Handler>
                 }
                 return;
             }
-            BMCWEB_LOG_CRITICAL("INFO: Reset OffloadUri of {} dump id {}",
-                                dumpType, entryID);
+            BMCWEB_LOG_INFO("INFO: Reset OffloadUri of {} dump id {}", dumpType,
```
Comment (jeaaustx) [inline include/dump_offload.hpp:188]: Doesn't the BMCWEB_LOG_INFO() already insert "INFO" into the message in the log? Seems like the "INFO:" can be removed from these messages now that they are not output as CRITICAL any more. (I.e. I think it was part of the string before to clarify that it wasn't actually a critical error.)

Comment (gtmills) [reply include/dump_offload.hpp:188]: Yeah, that is fair. 
I'll remove the INFO:

**Comment by raviteja-b** [comment]
I dont agree to remove these traces, all these traces are required to debug various issues which we have discussed earlier multiple times.

File: include/dump_offload.hpp (hunk 1 above)
Comment (raviteja-b) [reply include/dump_offload.hpp:188]: INFO logs can't be seen in bmc journal, there is no point in logging INFO traces

Comment (gtmills) [reply include/dump_offload.hpp:188]: INFO logs can be seen if you increase bmcweb logging level

Comment (gtmills) [reply include/dump_offload.hpp:188]: I added this to the agenda for Monday. 
I am going to wait here to remove the "INFO:" until we discuss

**Comment by gtmills** [comment]
Slack discussion at https://ibm-systems-power.slack.com/archives/G3WAVQYTH/p1726246893628949?thread_ts=1726239559.931879&cid=G3WAVQYTH

File: include/dump_offload.hpp (hunk 1 above)
Comment (gtmills) [reply include/dump_offload.hpp]: As discussed in slack, going to go with this

**Comment by gtmills** [comment]
> I dont agree to remove these traces, all these traces are required to debug various issues which we have discussed earlier multiple times.

Discussed in slack, we are going to go with this to fix this defect.

**Review by raviteja-b — APPROVED** [review]

**Pull request merged**